src/plugins_types.c

```diff
@@ -101,7 +102,12 @@ ly_xml_resolve_prefix(const struct ly_ctx *ctx, const char *prefix, size_t prefi
         return NULL;
     }
 
-    return ly_ctx_get_module_implemented_ns(ctx, ns->uri);
+    mod = ly_ctx_get_module_implemented_ns(ctx, ns->uri);
+    if (!mod) {
```

Comment:
is there a possibility to have it NULL even in another case than YIN extension prefix resolution? For example in case of some invalid data and using another way then would actually cause accepting the invalid input referencing some unexpected module here?

Reply:
I believe the only new situation that can happen is that a non-implemented module is returned but since `ly_schema_resolve_prefix` can do that, too, it should not be a problem.